Add glossary page for "registrable domain" instead of "eTLD" (#42588)
mdn/content: files/en-us/glossary/registrable_domain/index.md

---
title: Registrable domain
slug: Glossary/Registrable_domain
page-type: glossary-definition
sidebar: glossarysidebar
---

A **registrable domain** is a domain that can be (or has already been) registered by an individual or an organization through a domain name registrar. It's the highest level domain for a single {{glossary("site")}} on the web.

> Registrable domains are created directly underneath an _effective top-level domain_ (eTLD), such as `.org`, `.com`, or `.ac.uk`. For this reason a registrable domain is sometimes called an "eTLD+1".

Author: We could I suppose say something like "A registrable domain is created as a subdomain of an eTLD".

For example, all of the following are registrable domains:

```plain
crookedtimber.org
theguardian.com
sussex.ac.uk
```

> This means that all domains under each of these domains belong to the same organization. For example:

Reviewer: I'm not sure about "this means", because this does not follow ..

Mabye something like

```suggestion
All domains under a registrable domain belong to the same organization. For example:
```

Reviewer: Yes, but it reads like "Registrable domains are created directly underneath an _effective top-level domain_" is the _cause of_ "All domains under a registrable domain belonging to the same organisation". These are two facts, but they don't follow each other from what I can see. Anyway, that's why I commented.
